### ISSUE TITLE
[L3: SCREENREADER]: VET TEC - NVDA does not read the error if I forget to agree to the privacy policy 

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#1141]
## Issue
Migrated from original issue: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/17050

When I skip checking the **I have read the privacy policy** and press **Submit Application**, NVDA does not read out the error message. JAWS and VoiceOver both read it correctly. Screenshot attached below.

## Acceptance Criteria
As an NVDA + Firefox user, I want to understand that I must agree to the privacy policy before submitting my VET TEC application.

## Environment
* Win10 + Firefox
* NVDA
* https://staging.va.gov/education/apply-for-education-benefits/application/0994/review-and-submit

## Steps to Recreate
1. Log into the VET TEC form with test user
2. Answer questions and continue to Step 7: Review Application
3. Continue tabbing through the form until you get to the Submit Application button
4. Press `ENTER` or `SPACE` to submit, making sure you have not checked the required privacy policy
5. Verify the error message is never read out by NVDA

## Guidance from MDN
* https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_alert_role

## WCAG or Vendor Guidance (optional)
* [Labels or Instructions: Understanding SC 3.3.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/minimize-error-cues.html)
* [Error Identification: Understanding SC 3.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/minimize-error-identified.html)

---
![screen shot 2019-02-21 at 12 31 27 pm](https://user-images.githubusercontent.com/934879/53193032-983dbd80-35d5-11e9-9d5f-031cac489d10.png)

**Comment by k80bowman** [comment]
For this one, we'll need to check the design system checkbox component and make sure that NVDA is reading the error message.

**Comment by taylorkaren** [comment]
Issue moved to [department-of-veterans-affairs/va-forms-system-core #253](https://github.com/department-of-veterans-affairs/va-forms-system-core/issues/253) via [**ZenHub**](https://www.zenhub.com/)
